Script/char.dat

```diff
@@ -5517,7 +6289,16 @@ darkmage
       "@Dd is humming an incantation of a devastating death-spell.",
       "\"A mere mortal dares challenge my might?\"";
     }
-    FriendlyReplies == "\"Death awaits our foes.\"";
```

Comment:
Pretty sure Izzy was based on Tim the Enchanter from Monty Python's Holy Grail ("Death awaits ye, with nasty, big, pointy teeth"...)